corehq/apps/userreports/tests/test_report_filters.py

```diff
@@ -430,7 +430,7 @@ def test_pre_filter_value_exists(self):
             self.assertEqual(filter_value.to_sql_values(), {'exists_field_slug': ''})
             self.assertEqual(
                 str(filter_value.to_sql_filter().build_expression()),
-                'exists_field != :exists_field_slug OR exists_field IS NOT NULL'
+                'exists_field != :exists_field_slug'
```

Comment:
It took a bit of mental exercise for me to reason about why `NULL != ''` happens to do the right thing here. Maybe explicit is better than implicit in this case?

Reply:
I also wondered this, though it seemed unnecessarily complicated (also the OR should be an AND which I figured out in testing). Added comments in https://github.com/dimagi/commcare-hq/pull/28363 as a middle ground.